components/windows-service/HabService.cs

```diff
@@ -184,6 +185,16 @@ private void SupErrorHandler(object sender, DataReceivedEventArgs e)
         private void ExitHandler(object sender, System.EventArgs e)
         {
             log.Error(String.Format("Habitat Supervisor has exited with exit code {0}", proc.ExitCode));
+            var mos = new ManagementObjectSearcher("SELECT ProcessId FROM Win32_Process WHERE Name='hab-sup.exe' AND ParentProcessId=" + proc.Id);
```

Comment:
Wait, you get to search for processes _using SQL_?!

![](https://media.giphy.com/media/3oEjHXTdZtYRmhWzny/giphy.gif)

Reply:
everything is a database!